src/main/java/resources/PersonsResource.java

```diff
@@ -84,17 +84,9 @@ public Notes getNote(@PathParam("personId") Long personId,
     @Consumes(MediaType.APPLICATION_JSON)
     @UnitOfWork
     public Response createOrUpdate(Notes note){
-        Date now = new Date();
-        Calendar c = Calendar.getInstance();
-        c.setTime(note.getDate());
-        c.add(Calendar.DATE, 5); // Adding 5 days
-        Date comparisonDate = new Date(c.getTimeInMillis());
-        if (now.after(comparisonDate)) { // don't allow users to submit availabilities older than current week
-            return Response.status(Response.Status.NOT_ACCEPTABLE).build();
-        } else {
-            notesDao.createOrUpdate(note);
-            return Response.status(Response.Status.ACCEPTED).entity(note).build();
-        }
```

Comment:
We use a version control system for a reason -- nothing gets lost :) Please remove commented out code.